### ISSUE TITLE
More natural structures in OmniMap

### ISSUE THREAD

**Issue by maximusvladimir** [JPII/NavalBattle#73]
I would like to see:
1. Caves.
2. Grass.
3. Trees.
4. Anmals.
5. Baby Whales (their venom is stronger than a normal whale, because it is more concentrated).
6. Detailed ships.
7. Villages.

(It's going to look like Minecraft with an HD texture pack by the time I am done with it).

**Comment by ThomasGaubert** [comment]
`Minecraft with an HD`
I never liked MC with an HD pack ;)

Sounds fine though, but we should get a semi-working game first.

**Comment by maximusvladimir** [comment]
Yeah, this is just an idea, I don't plan on adding this stuff anytime soon, which is why it is bumped to Beta 0.7.

**Comment by ThomasGaubert** [comment]
Okay, cool.

**Comment by abauer** [comment]
Definitely think this needs to  move to Beta 0.8, its a feature

**Comment by ThomasGaubert** [comment]
Moved.

**Comment by maximusvladimir** [comment]
:frowning:

**Comment by maximusvladimir** [comment]
Honestly, I am somewhat upset at the fact that many issues are being closed that have not yet been achieved, and they are in future milestones.

In my opinion, when the time comes for these issues to apply for the current milestone, we can either bump it, or come to a mutual understanding about changing/removing an issue.

**Comment by abauer** [comment]
agree in part about the re-evaluate during the milestone... but i think this one can stay closed... None of the above-mentioned goals provide functionality

**Comment by maximusvladimir** [comment]
"but i think this one can stay closed... "

Ummm well this is a feature of Pavo, and I understand that it doesn't provide functionality for the game, but when I do move Pavo out of NavaBattle, I would also like to move issues that weren't completed either.

The point I was trying to make in my previous comment, is that instead of closing issues that we no longer "want", I think we should move them to like 1.1, so if we do want to implement them later, we don't forget about them. I understand that this particular issue is not part of this case.

**Comment by abauer** [comment]
agree, moving them to 1.1. is a viable solution;

(just saying, this SHOULD NOT BE DONE NOW)
Another solution to this would have been to split pavo into a separate repo so that these kind of issues could be placed there...
(its too late now)

**Comment by maximusvladimir** [comment]
"(just saying, this SHOULD NOT BE DONE NOW)"

Trust me, I don't plan on it.

**Comment by abauer** [comment]
ok good
